spotify_project/artists/urls.py

```diff
@@ -5,6 +5,10 @@
 app_name = 'artists'
 
 urlpatterns = [
+    re_path(
+        r'^my_artists/$', views.all_artists, name='my_artists', kwargs={'mine': True}
```

Comment:
Looks like the branch was branched off another branch. Rather then off `master`.

Reply:
It was branched off of `refactor_spotify_data_pull`. My thought was that I could never merge that other PR, since it broke all the tests. So, I made this branch of off that one to make the tests. Then, I can merge this branch to that one, and the tests will be passing.